src/main/kotlin/org/jitsi/nlj/RtpSenderImpl.kt

```diff
@@ -113,11 +114,9 @@ class RtpSenderImpl(
     private val nackHandler: NackHandler
 
     /**
-     * The packet loss to introduce in the send pipeline (for debugging/testing purposes).
+     * Configuration for the packet loss to introduce in the send pipeline (for debugging/testing purposes).
      */
-    private val lossFractionToIntroduce: Double by config {
-        "jmt.debug.packet-loss.send".from(JitsiConfig.newConfig)
-    }
+    private val packetLossConfig = PacketLossConfig("jmt.debug.packet-loss.outgoing")
```

Comment:
I think we've avoided multiple instances of the same config class in fear of out-of-sync problems.  This means we'd have to expose a static instance of this, which isn't great...but I worry about mismatched weirdness of loading them twice.

Reply:
I can make this static in RtpSenderImpl